src/util/string.h

```diff
@@ -15,6 +16,8 @@
 #include <string>
 #include <vector>
 
+constexpr auto SplitString = &spanparsing::Split<std::string>;
```

Comment:
I think it would be safer to use a more restrictive interface for `SplitString`, because it's not really intuitive how SplitString behaves with string literals. Something like

```cpp
inline std::vector<std::string> SplitString(std::string_view str, char sep) {
    return spanparsing::Split<std::string>(str, sep);
}
```